packages/core/contracts/financial-templates/common/TokenFactory.sol

```diff
@@ -22,7 +22,7 @@ contract TokenFactory is Lockable {
         string calldata tokenName,
         string calldata tokenSymbol,
         uint8 tokenDecimals
-    ) external nonReentrant() returns (ExpandedIERC20 newToken) {
+    ) external virtual nonReentrant() returns (ExpandedIERC20 newToken) {
```

Comment:
This is made virtual so that `TokenFactoryExclusiveMinter` can overwrite the `createToken` method.